core/src/test/java/org/axonframework/commandhandling/CommandHandlerInterceptorTest.java

```diff
@@ -153,7 +155,8 @@ public void testInterceptorWithNonVoidReturnType() {
 
     @Test
     public void testInterceptorWithDeclaredChainAllowedToDeclareNonVoidReturnType() {
-        new EventSourcingRepository<>(MyAggregateWithDeclaredInterceptorChainInterceptorReturningNonVoid.class, eventStore);
+        new EventSourcingRepository<>(MyAggregateWithDeclaredInterceptorChainInterceptorReturningNonVoid.class,
```

Comment:
What does this test actually test? 
Will it throw an exception when the creation of a `EventSourcingRepository` fails?

Reply:
Yes, it will throw an exception if that fails. Does so because it can't instantiate an `AggregateModel` from the provided `aggregateType` (which is the `MyAggregateWithDeclaredInterceptorChainInterceptorReturningNonVoid.class` object).